package.json

```diff
@@ -0,0 +1,10 @@
+{
+  "devDependencies": {
+    "@types/node": "^20.10.0"
+  },
+  "dependencies": {
+    "dotenv": "^16.3.1",
+    "fs": "^0.0.1-security",
+    "openai": "^4.20.1"
+  }
+}
```

Comment:
These should be inside of the typescript folder's package.json right? (and then you don't need package-lock.json or yarn.lock in the root folder